lapis-e2e/test/alignedNucleotideSequence.spec.ts

```diff
@@ -14,7 +13,12 @@ describe('The /alignedNucleotideSequence endpoint', () => {
     });
 
     expect(result).to.have.length(100);
-    expect(result[0].primaryKey).to.equal('key_3259931');
+    result.sort((a: any, b: any) => {
```

Comment:
Explicit `any` is even worse that implicit `any`. It simply kills all type information (whereas having no types leaves it open to be inferred).
```suggestion
    result.sort((a, b) => {
```

Reply:
I need to add a type or I get a linter error

Reply:
What linter error? I don't think we have a linter in those tests. Then let's add something that's more useful (I also noticed that our general TS setup is not really good here and there are places where tsc complains, but unfortunately we're not running it in the pipeline...)

Reply:
I get the error: "Parameter 'a' implicitly has an 'any' type" which is a TypeScript error - Specifically, it’s a TypeScript compiler error about missing type annotations.

Reply:
If you have that here you'll also have it elsewhere in the tests I guess. I'd suggest to ignore it for now, this is not an issue that we need to solve in this PR. The actual problem is somewhere else since `result` already seems to have missing type infos (if `result` had proper types, TS would be able to infer the types of `a` and `b` here).